tests/unit/introspection.tcl

```diff
@@ -402,3 +402,30 @@ start_server {tags {"introspection"}} {
     # Config file at this point is at a weird state, and includes all
     # known keywords. Might be a good idea to avoid adding tests here.
 }
+
+test {config during loading} {
+    start_server [list overrides [list key-load-delay 50 rdbcompression no]] {
+        # create a big rdb that will take long to load. it is important
+        # for keys to be big since the server processes events only once in 2mb.
+        # 100mb of rdb, 100k keys will load in more than 5 seconds
+        r debug populate 100000 key 1000
```

Comment:
This seems ok, but 100mb is a lot and we also can't be sure it'll really take 5 seconds. Don't we have something like `rdb-key-save-delay` for loading? Should we add one?

Reply:
it does use `key-load-delay 50` and `rdbcompression no`.
i copied this code from another test...
100mb is indeed a lot, but in theory 5 seconds is what we need to avoid race condition on slow machines.
note that we don't wait for it to load, we kill the server when we're done with the check.

Reply:
Ok, got it. Because of the 2mb processing interval we need a huge rdb. The load delay isn't enough on its own.